<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_book</class>
 <widget class="QWidget" name="edit_book">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>771</width>
     <height>421</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QPushButton" name="search_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>search</string>
     </property>
    </widget>
    <widget class="QListWidget" name="Book_list">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>60</y>
       <width>321</width>
       <height>320</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="search">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>421</width>
       <height>281</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="Author_label">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>72</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Author</string>
      </property>
     </widget>
     <widget class="QLabel" name="Name_label">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>28</y>
        <width>36</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Namebook">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>28</y>
        <width>261</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Author">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>72</y>
        <width>261</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="genre">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>118</y>
        <width>261</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="gen_lable">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>118</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Genre</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addnew">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>210</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add New</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QSpinBox" name="Number">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>160</y>
        <width>47</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="No_label">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>160</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QListWidget" name="Book_list_edit">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>60</y>
       <width>321</width>
       <height>320</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_edit">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>421</width>
       <height>281</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="Author_label_edit">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>72</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Author</string>
      </property>
     </widget>
     <widget class="QLabel" name="Name_label_edit">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>28</y>
        <width>36</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Namebook_edit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>28</y>
        <width>261</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Author_edit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>72</y>
        <width>261</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="genre_edit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>108</y>
        <width>261</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="gen_lable_edit">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>108</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Genre</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Update">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>220</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QSpinBox" name="Number_edit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>47</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="No_label_edit">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>150</y>
        <width>49</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Total </string>
      </property>
     </widget>
     <widget class="QPushButton" name="remove">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>220</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="No_label_edit_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>150</y>
        <width>49</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Left</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="number_left_edit">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>150</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="search_edit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QListWidget" name="re_name">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>321</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="re_status">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>60</y>
       <width>131</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="re_acc">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>60</y>
       <width>161</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>130</y>
       <width>111</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="accept">
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Accept</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="deny">
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Deny</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>Book_list</tabstop>
  <tabstop>search</tabstop>
  <tabstop>Namebook</tabstop>
  <tabstop>Author</tabstop>
  <tabstop>remove</tabstop>
  <tabstop>genre</tabstop>
  <tabstop>Number</tabstop>
  <tabstop>addnew</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
